File: swagger/models/add_client_direct_debit_info_request.py
from .base_model import BaseModel


class AddClientDirectDebitInfoRequest(BaseModel):
	"""
	no description available

	Attributes:

		test
		A value of true performs a dry run exectuion

		client_id
		The ID of the client being updated

		name_on_account
		The name on the bank account being added

		routing_number
		The routing number of the bank account being added

		account_number
		The account number of the bank account being added

		account_type
		Either "Checking" or "Savings"
	"""

	input_map = {
		'Test': 'test',
		'ClientId': 'client_id',
		'NameOnAccount': 'name_on_account',
		'RoutingNumber': 'routing_number',
		'AccountNumber': 'account_number',
		'AccountType': 'account_type',
		}

	output_map = {
		'test': 'Test',
		'client_id': 'ClientId',
		'name_on_account': 'NameOnAccount',
		'routing_number': 'RoutingNumber',
		'account_number': 'AccountNumber',
		'account_type': 'AccountType',
		}

	test = None
	client_id = None
	name_on_account = None
	routing_number = None
	account_number = None
	account_type = None

File: swagger/Models/AddClientDirectDebitInfoRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class AddClientDirectDebitInfoRequest extends BaseModel
{
	/**
	 * A value of true performs a dry run exectuion
	 * 
	 * @var boolean 
	 */
	public $test;
	/**
	 * The ID of the client being updated
	 * 
	 * @var string 
	 */
	public $clientId;
	/**
	 * The name on the bank account being added
	 * 
	 * @var string 
	 */
	public $nameOnAccount;
	/**
	 * The routing number of the bank account being added
	 * 
	 * @var string 
	 */
	public $routingNumber;
	/**
	 * The account number of the bank account being added
	 * 
	 * @var string 
	 */
	public $accountNumber;
	/**
	 * Either "Checking" or "Savings"
	 * 
	 * @var string 
	 */
	public $accountType;

	protected function getInputMap()
	{
		return [
			'Test' => ['test', 'boolean', null],
			'ClientId' => ['clientId', 'string', null],
			'NameOnAccount' => ['nameOnAccount', 'string', null],
			'RoutingNumber' => ['routingNumber', 'string', null],
			'AccountNumber' => ['accountNumber', 'string', null],
			'AccountType' => ['accountType', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'test' => ['Test', 'boolean', null],
			'clientId' => ['ClientId', 'string', null],
			'nameOnAccount' => ['NameOnAccount', 'string', null],
			'routingNumber' => ['RoutingNumber', 'string', null],
			'accountNumber' => ['AccountNumber', 'string', null],
			'accountType' => ['AccountType', 'string', null],
			];
	}
}
